logic/scorer/scorer.go

```diff
@@ -15,6 +15,23 @@ func CribPoints(lead model.Card, crib []model.Card) int {
 	return points(lead, crib, true)
 }
 
+const cardsPerHand = 5
+
+// map card value (1 to 13) to number of occurrences in the hand
+// we have to include index 0 and index 14 here because some calculations pass those values in
+type valueToCount [15]uint8
+
+func (vtc valueToCount) get(value int) (uint8, bool) {
+	return vtc[value], vtc.containsKey(value)
+}
+
+func (vtc valueToCount) containsKey(key int) bool {
+	return vtc[key] > 0
+}
```

Comment:
:mage: Paired with another challenge later on, can we get away with _not_ defining these methods on this type?